engine/src/main/java/com/datatorrent/stram/LaunchContainerRunnable.java

```diff
@@ -232,7 +232,7 @@ public void run()
 
     String loggerAppender = dag.getValue(LogicalPlan.LOGGER_APPENDER);
     if (loggerAppender != null) {
-      vargs.add(String.format("-D%s=\"%s\"", LogicalPlan.LOGGER_APPENDER.getLongName(), loggerAppender));
+      vargs.add(String.format("-D%s='%s'", LogicalPlan.LOGGER_APPENDER.getLongName(), loggerAppender));
```

Comment:
Fixed.

"$" is not a special symbol inside single quotes in bash. But it is a special symbol inside double quotes.

[sergey@node0 ~]$ echo 'test- $NAME -test'
test- $NAME -test
[sergey@node0 ~]$ echo "test- $NAME -test"
test-  -test